### DISCUSSION TITLE
Make kutt private (running on Unraid in docker)

### DISCUSSION THREAD

**Discussion by dratze98** [thedevs-network/kutt#707] [Q&A] [answered]
Hello,
How do I make it so that you have to login to shorten a url and than public sign up isn't allowed?
I found that I can set:
DISALLOW_ANONYMOUS_LINKS=true
DISALLOW_REGISTRATION=true
But when I do that via variables in the docker template, it doesn't work. Where do I need to set those?

Thanks

**Comment by dratze98** [comment, marked as answer]
Nevermind. I am dumb. I didn't see the already present fields for these variables and was adding my own. It is working now.

**Reply by qplus7** [reply]
I am trying to get Kutt running on UnRaid as well. What did you do to enable to sign up option? I'm running into a brick wall here... thanks in advance!

**Reply by dratze98** [reply]
Hey sorry for not seeing this sooner. Just happened to come back and see your question. What solved my issue was clicking "Show more settings..." in the docker container edit page when installing. There are two variables DISALLOW_REGISTRATION and DISALLOW_ANONYMOUS_LINKS which I set to false and true respectively. This way users are able to make accounts and they cannot create links without having an account.

Hope this helps! Let me know if this doesn't solve your issue, and I can try to help.
